src/main/java/org/jenkinsci/maven/plugins/hpi/InitializeMojo.java

```diff
@@ -22,13 +29,14 @@
  *
  * @author Basil Crow
  */
-@Mojo(name = "initialize", defaultPhase = LifecyclePhase.INITIALIZE)
+@Mojo(name = "initialize", requiresDependencyResolution = ResolutionScope.TEST, defaultPhase = LifecyclePhase.INITIALIZE)
```

Comment:
I think it would make more sense to do this in a test preparation phase if we can?

Reply:
I agree. I might split this out into a different mojo soonish in some separate PR.